facia/test/FaciaControllerTest.scala

```diff
@@ -140,12 +140,6 @@ import controllers.FaciaControllerImpl
     faciaController.alternativeEndpoints("uk/business/stock-markets") should be (List("business", "uk"))
   }
 
-  it should "render fronts with content that has been pre-fetched from facia-press" in {
-    // make sure you switch on facia inline-embeds switch before you press to make this pass
-    val request = FakeRequest("GET", "/inline-embeds").from(Uk)
```

Comment:
@johnduffell @rich-nguyen I deleted this test as the `/inline-embeds` front no longer exists on CODE, so I cannot generate the test data to make this test pass anymore. If this test is important to you, please publish some test data on CODE to make it pass.